server/src/test/java/org/elasticsearch/index/mapper/NumberFieldMapperTests.java

```diff
@@ -218,6 +218,13 @@ public void testOutOfRangeValues() throws IOException {
                     e.getCause().getMessage(), containsString(item.message));
             }
         }
+    }
 
+    @Override
+    protected final Object generateRandomInputValue(MappedFieldType ft) {
+        Number n = randomNumber();
+        return randomBoolean() ? n : n.toString();
```

Comment:
Is this going to run into #69382 and the issues with scientific format? Or does that only apply to date parsing?

Reply:
Numbers are fine parsing scientific notation. It's dates that hate 'em. There is an issue with buried in the somewhere but I've awaitsFixed that bit and opened another issue.